cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -111,18 +111,15 @@ func parsePluginConfig(pluginConfigPath string) VersionsInSummary {
 	if err != nil {
 		// return default value of VersionsInSummary
 		return VersionsInSummary{MajorVersionsInSummary,
-			MinorVersionsInSummary, PatchVersionsInSummary}
+			MinorVersionsInSummary, PatchVersionsInSummary}, nil
 	}
 	err = json.Unmarshal([]byte(pluginConfig), &config)
 	if err != nil {
-		// return default value of VersionsInSummary
-		return VersionsInSummary{MajorVersionsInSummary,
-			MinorVersionsInSummary, PatchVersionsInSummary}
+		return VersionsInSummary{}, status.Errorf(codes.InvalidArgument, "plugin-config-path: %s error: %v", string(pluginConfig), err)
```

Comment:
Note that we parse configs during startup, not when handling grpc requests, so we have no need for grpc related status codes via `status.Errorf` (it is specifically for associating an error code with a grpc request/response cycle). You can simply use `fmt.Errorf("failed to unmarshal plugin config: %w", err)` which wraps the returned error with the specific message that's relevant here (which you then correctly log with `log.Fatalf` below).

Reply:
used 'errors' and removed Code.